StandardLibrary/runtimes/go/ImplementationFromDafny-go/Time_/externs.go

```diff
@@ -38,7 +38,10 @@ func (CompanionStruct_Default___) GetProcessCpuTimeMillis() int64 {
 }
 
 func GetProcessCpuTimeMillis() int64 {
-	usage := new(syscall.Rusage)
-	syscall.Getrusage(syscall.RUSAGE_SELF, usage)
+	var usage syscall.Rusage
+	err := syscall.Getrusage(syscall.RUSAGE_SELF, &usage)
+	if err != nil {
+		return 0
```

Comment:
I assume 0 is okay, but why not return error in the method signature?

Reply:
Because what else could the caller do but pretend it was zero?
None of the other languages had this issue, and so like "get current time" it seems silly to return an error.

Reply:
Yeah, Go returns error with output instead of separately? Maybe `-1` if not err?